src/components/tab-title/tab-title.e2e.ts

```diff
@@ -50,6 +84,241 @@ describe("calcite-tab-title", () => {
+      expect(containerElOne).toHaveAttribute(HYDRATED_ATTR);
+
+      await closeOne.click();
+      await page.waitForChanges();
+
+      containerElOne = await page.find(`calcite-tab-title[id='one']>>> .${CSS.container}`);
+      expect(await containerElOne.getProperty("hidden")).toBe(true);
+
+      const closeTwo = await page.find(`calcite-tab-title[id='two'] >>> .${CSS.closeButton}`);
+      expect(await closeTwo.getProperty("closable")).not.toBe(true);
+    });
+  });
+
+  describe("closing sequence", () => {
+    let page: E2EPage;
+
+    let matchingTabEl: E2EElement;
+    let close: E2EElement;
+
+    const closeTabsInSequenceOfGivenArrayOfIds = async (arrayOfIds: string[]): Promise<void> => {
+      for (let i = 0; i < arrayOfIds.length - 1; i++) {
+        let tabEl: E2EElement;
+        let tabTitleContainerEl: E2EElement;
+
+        const id = arrayOfIds[i];
+
+        tabEl = await page.find(`#${id}`);
+        tabTitleContainerEl = await page.find(`calcite-tab-title[id='${id}'] >>> .${CSS.container}`);
+        const close = await page.find(`calcite-tab-title[id='${id}'] >>> .${CSS.closeButton}`);
+
+        expect(await tabTitleContainerEl.getProperty("hidden")).not.toBe(true);
+
+        await close.click();
+        await page.waitForChanges();
+
+        const tabTitleEl = await page.find(`#${id}`);
+        tabEl = await page.find(`#${id}`);
+        tabTitleContainerEl = await page.find(`calcite-tab-title[id='${id}'] >>> .${CSS.container}`);
+
+        expect(await tabTitleContainerEl.getProperty("hidden")).toBe(true);
+        expect(await tabTitleEl.getProperty("selected")).not.toBe(true);
+        expect(await tabEl.getProperty("selected")).not.toBe(true);
+
+        const nextId = arrayOfIds[i + 1];
+        const nextTabTitleEl = await page.find(`#${nextId}`);
+        const nextTabEl = await page.find(`#${nextId}`);
+        const nextTabTitleContainerEl = await page.find(`calcite-tab-title[id='${nextId}'] >>> .${CSS.container}`);
+
+        expect(await nextTabTitleContainerEl.getProperty("hidden")).not.toBe(true);
+        expect(await nextTabTitleEl.getProperty("selected")).toBe(true);
+        expect(await nextTabEl.getProperty("selected")).toBe(true);
+      }
+    };
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(multiTabTitleClosableHtml);
+    });
+
+    it(`when closing tab-titles in sequence 1 (first selected) through 4, 
+        tab-title and corresponding tab become hidden, 
+        and selection fallback is the next tab`, async () => {
+      await closeTabsInSequenceOfGivenArrayOfIds(["embark", "car", "plane", "biking"]);
+    });
+
+    it(`when closing tab-titles in sequence 4 (last selected) through 1, 
+        tab-title and corresponding tab become hidden, 
+        and selection fallback is the previous tab`, async () => {
+      const arrayOfReversedIds = ["embark", "car", "plane", "biking"].reverse();
```

Comment:
Is there a more expressive name for this? What do these IDs represent? Applies to other ID arrays.